tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/NewLiferayComponentServiceNameValidataionService.java

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.core.modules;
+
+import com.liferay.ide.core.util.CoreUtil;
+
+import org.eclipse.sapphire.FilteredListener;
+import org.eclipse.sapphire.Listener;
+import org.eclipse.sapphire.PropertyContentEvent;
+import org.eclipse.sapphire.modeling.Status;
+import org.eclipse.sapphire.services.ValidationService;
+
+/**
+ * @author Simon Jiang
+ */
+
+public class NewLiferayComponentServiceNameValidataionService extends ValidationService
```

Comment:
This service is unneeded you can simply just do a "@Required" and it will do the same thing as this class which is just checking if the service exists or not.